src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleChooserControl.xaml

```diff
@@ -378,7 +378,7 @@
 											   Margin="{StaticResource NavIconMargin}" />
 								<TextBlock Style="{StaticResource Typo06}"
 										   VerticalAlignment="Bottom"
-										   Text="FAVORITES" />
+										   Text="FAVS" />
```

Comment:
?

Reply:
It's breaking the design on iOS and Android because it's too long and forcing a line break.